.gitlab/benchmarks/macrobenchmarks.yml

```diff
@@ -394,6 +394,177 @@ profiler_cpu_timer_create-arm64:
   variables:
     NATIVE_PROFILER_PATH: "dd-trace-dotnet/tracer/tracer-home-linux-arm64/linux-arm64/Datadog.Trace.ClrProfiler.Native.so"
     TRACER_HOME_PATH: "dd-trace-dotnet/tracer/tracer-home-linux-arm64"
+    COR_ENABLE_PROFILING: 1
+    CORECLR_ENABLE_PROFILING: 1
+    DD_PROFILING_ENABLED: 1
+    DD_PROFILING_WALLTIME_ENABLED: 0
+    DD_PROFILING_CPU_ENABLED: 1
+    COMPlus_EnableDiagnostics: 1
+    ENDPOINT: "hello"
+
+.benchmarks-win:
+  stage: benchmarks-win
+  needs: ["check_azure_pipeline"]
+  tags: ["runner:docker"]
+  interruptible: true
+  timeout: 1h
+  # Image based on earlier version of tooling
+  image: registry.ddbuild.io/images/benchmarking-platform-tools-ubuntu:dotnet-tests
+
+  script:
+    - export DD_API_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.$CI_PROJECT_NAME.dd_api_key --with-decryption --query "Parameter.Value" --out text)
+    - CLEANUP_ARG=$([[ "$CLEANUP" == "false" ]] && echo "--no-cleanup" || echo "")
+    - mkdir -p ~/.aws
+    - /app/infra/tools/fetch-ssm-parameter.sh $AWS_EPHEMERAL_INFRA_PROFILE_SSM_PARAMETER \
+        >> ~/.aws/config || exit $?
+    - |
+      aws ssm get-parameter --region "$AWS_REGION" --name "ci.${CI_PROJECT_NAME}.ephemeral-infra-ci.windows-benchmarking-key-pair-name" --with-decryption --query "Parameter.Value" --out text >> ~/.aws/key-pair-name.txt
+      aws ssm get-parameter --region "$AWS_REGION" --name "ci.${CI_PROJECT_NAME}.ephemeral-infra-ci.windows-benchmarking-key-private-key" --with-decryption --query "Parameter.Value" --out text >> ~/.aws/key-pair-private-key.pem
+    - export AWS_PROFILE=ephemeral-infra-ci
+    - export BP_INFRA_KEY_PAIR_NAME=$(cat ~/.aws/key-pair-name.txt)
+    - export BP_INFRA_KEY_PAIR_PRIVATE_KEY_PATH=~/.aws/key-pair-private-key.pem
+    - git clone --branch fayssal/migrate-macros-dotnet https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/benchmarking-platform platform
+    - ./platform/pass-artifatcs-to-windows.sh
```

Comment:
>./platform/pass-artifatcs-to-windows.sh

Typo in script name.

Reply:
Thank you, actually it's a typo in the name of the file downstream itself 😅 (hence why the tests are successful), I'll change the name of the file and thus the name in the script!